box_layout.kv

```diff
@@ -1,9 +1,15 @@
 BoxLayout:
```

Comment:
Thanks for looking at it for me.

________________________________
From: DavidHouse369 <notifications@github.com>
Sent: Saturday, September 10, 2016 5:08:13 PM
To: garethmullins/Python-Week6-Practicals
Cc: Gareth Mullins; Author
Subject: Re: [garethmullins/Python-Week6-Practicals] Code review request (#1)


In box_layout.kv<https://github.com/garethmullins/Python-Week6-Practicals/pull/1#discussion_r78272875>:

> @@ -1,9 +1,15 @@
>  BoxLayout:


Everything looks good

-
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/garethmullins/Python-Week6-Practicals/pull/1/files/eaf23b31a58ec87253d8fe9096a2ed84446b898a#r78272875>, or mute the thread<https://github.com/notifications/unsubscribe-auth/ARNMdQzsVu9y1L83hVUs7sYNrlz5BAIYks5qolddgaJpZM4J5Ao_>.